tests/rbd/rbd_utils.py

```diff
@@ -40,7 +47,7 @@ def __init__(self, **kw):
                 if self.config.get("ec_pool_config", {}).get("data_pool")
                 else "rbd_test_data_pool_" + self.random_string()
             )
-            self.k_m = "2,1"
+            self.k_m = self.config.get("ec-pool-k-m", "2,1")
```

Comment:
`self.k_m = "2,1" if not self.k_m else self.k_m`

Reply:
```
if not self.k_m:
   self.k_m = "2,1"
```

Reply:
done